infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationUtils.kt

```diff
@@ -107,6 +110,8 @@ private fun formatChangedKmNumbers(kmNumbers: List<KmNumber>) =
         acc
     }.joinToString(", ") { it.joinToString("-") }
 
+fun formatDistance(dist: Double) = if (dist >= 0.1) "${roundTo1Decimal(dist)}" else "<${roundTo1Decimal(0.1)}"
```

Comment:
Täällä toteutettu erikoiskäsittely sille, jos muutos on tarpeeksi merkittävä että se näkyy arvoissa, mutta se on kumminkin alle 10cm. Tällöin arvoksi tulee "<0.1"